### DISCUSSION TITLE
I keep getting Event Handler exceptions after updating from 4.2.0 to 4.3.0

### DISCUSSION THREAD

**Discussion by samjesus8** [DSharpPlus/DSharpPlus#1456] [Q&A] [answered]
I updated my whole solution from Version 4.2.0 to 4.3.0. I didn't change anything in my code and i just ran the Bot only to get swarmed with these errors below

![image](https://user-images.githubusercontent.com/98812930/213891857-dfa691fc-57f8-4884-ab56-b492b423b7b1.png)

My commands are still sending messages but I don't know if this error is something that might affect the bot so can someone please tell me how to sort this out

![image](https://user-images.githubusercontent.com/98812930/213892002-5042e46d-ad8a-427d-8f1a-cc56ab95cc36.png)

**Comment by akiraveliara** [comment, marked as answer]
Judging off the error, make sure you have ConcurrentHashSet 1.1.0 installed, no later. You might have to explicitly reference 1.1.0 in your project file.

**Reply by samjesus8** [reply]
Wow, yeah I clicked on everything and updated all so I didn't realize that was the issue. Well that fixed it anyway lmaoooo
Yeah it updated ConcurrentHashSet to 1.3.0 so i just downgraded and its all good
